tests/testapp/ui.json

```diff
@@ -1988,7 +1988,7 @@
                 "eventType": "ss-change",
                 "stateRef": "name"
             },
-            "visible": ""
```

Comment:
This seems to be a typo: I've verified that actual UI update resolves to "false" for components with disabled visibility. And, this failed Pydantic validation, so I had to change it.

Reply:
As it turned out, wasn't a typo – "visible" property may be a string in case of custom visibility. Thank you @ramedina86 for pointing this out!